feat: Show active file name when saving to current file

h7y opened the pull request:
## Screenshots
![Screenshot 2021-06-13 at 12 08 54 PM](https://user-images.githubusercontent.com/22396000/121797798-32f7b580-cc40-11eb-8d2f-8957ad0bcd44.png)

### Long file name
![Screenshot 2021-06-13 at 12 08 33 PM](https://user-images.githubusercontent.com/22396000/121797801-3428e280-cc40-11eb-9974-6a2e07f62f23.png)


## Closes
#3650

dwelle commented:
Notes:

1. I'd not make it into an input at all since that suggests editability. I'm not technically against making it editable, but would leave that for later as we'd need to figure out how it should behave — rename the file (if possible)? Save to a new file handle?

2. The shape props menu should account for the fileHandle and not overflow the footer:
	
	![image](https://user-images.githubusercontent.com/5153846/121800314-65e67d00-cc31-11eb-948f-acf54a0f77a5.png)

h7y commented:
![Screenshot 2021-06-13 at 4 11 57 PM](https://user-images.githubusercontent.com/22396000/121804087-4b78c780-cc62-11eb-9c5c-c4aa05d127f6.png)
![Screenshot 2021-06-13 at 4 11 27 PM](https://user-images.githubusercontent.com/22396000/121804088-4c115e00-cc62-11eb-8ecd-42a85566fcd0.png)

h7y commented:
1) I've changed it more to look like the UI from #3381 
2) Fixed it by changing the max height of shape props menu depending on whether active file name is shown

ad1992 commented on src/components/ActiveFile.tsx:
```diff
@@ -0,0 +1,29 @@
+
+type ActiveFileProps = {
+  fileName?: string;
+  onClick: () => void;
```
lets rename this to be `onSave` instead to be more clear

ad1992 commented on src/components/ActiveFile.tsx:
```diff
@@ -0,0 +1,29 @@
+import "./ActiveFile.scss";
+
+import React from "react";
+import Stack from "../components/Stack";
+import { ToolButton } from "../components/ToolButton";
+import { save, file } from "../components/icons";
+import { t } from "../i18n";
```
```suggestion
import React from "react";
import Stack from "../components/Stack";
import { ToolButton } from "../components/ToolButton";
import { save, file } from "../components/icons";
import { t } from "../i18n";

import "./ActiveFile.scss";
```

For consistency, as we are habituated seeing the first line `import React...` :)

ad1992 reviewed (commented):
Overall LGTM 👍🏻

dwelle commented:
I'd tweet it alongside https://github.com/excalidraw/excalidraw/pull/3717 once merged

ad1992 commented:
shipping 🎉
